### ISSUE TITLE
FIO plugin failed to find OCF bdev in CI

### ISSUE THREAD

**Issue by shuhei-matsumoto** [spdk/spdk#1241]
https://ci.spdk.io/results/autotest-per-patch/builds/5030/archive/nvme-phy-autotest/build.log
```
Starting 5 threads
Starting SPDK v20.04-pre git sha1 c96caeb57 / DPDK 19.11.0 initialization...
[ DPDK EAL parameters: fio --no-shconf -c 0x1 --log-level=lib.eal:6 --log-level=lib.cryptodev:5 --log-level=user1:6 --base-virtaddr=0x200000000000 --match-allocations --file-prefix=spdk_pid175619 ]
EAL: No available hugepages reported in hugepages-1048576kB
EAL: VFIO support initialized
fio_plugin.c: 441:spdk_fio_setup: *ERROR*: Unable to find bdev with name PT_Nvme
fio_plugin.c: 441:spdk_fio_setup: *ERROR*: Unable to find bdev with name PT_Nvme
fio_plugin.c: 441:spdk_fio_setup: *ERROR*: Unable to find bdev with name PT_Nvme
fio_plugin.c: 441:spdk_fio_setup: *ERROR*: Unable to find bdev with name PT_Nvme
fio_plugin.c: 441:spdk_fio_setup: *ERROR*: Unable to find bdev with name PT_Nvme

```

Run status group 0 (all jobs):

**Comment by jimharris** [comment]
Bug scrub: This appears OCF related.  Assigning to @mdziegie

**Comment by Donaim** [comment]
I will investigate the error this week. There was no new changes in OCF since like half a year, so this issue must be really rare.

**Comment by Donaim** [comment]
Probably related to https://github.com/spdk/spdk/issues/1287

**Comment by Donaim** [comment]
Probably related to #1287 because I get the same error message when using json config

**Comment by Donaim** [comment]
I proposed a fix to #1287 which should apply to this issue as well

**Comment by Donaim** [comment]
I couldn't reproduce this manually, but I am sure that my fix for #1287 should apply to this as well
So I think we should close it if it doesn't reproduce anymore after the fix is merged

**Comment by Donaim** [comment]
Fix to #1287 was merged, so if this issue doesn't reproduce anymore, I think we should close it

**Comment by Donaim** [comment]
No new reports for more than a week now, so closing this.
Please reopen if needed
